addon/helpers/record-meta.js

```diff
@@ -0,0 +1,47 @@
+import Helper from '@ember/component/helper'
+import { action as bind } from '@ember/object'
+import { inject as service } from '@ember/service'
+import { EVENT } from 'ember-data-record-meta/config'
+
+export default class RecordMetaHelper extends Helper {
+  /**
+   * Services
+   */
+
+  @service('record-meta') recordMetaService
+
+  /**
+   * Life cycle
+   */
+
+  constructor () {
+    super(...arguments)
+
+    this.recordMetaService.on(
+      EVENT.RECORD_META_CHANGED,
+      this.recordMetaChangedHandler
+    )
+  }
+
+  willDestroy () {
+    super.willDestroy(...arguments)
+
+    this.recordMetaService.off(
+      EVENT.RECORD_META_CHANGED,
+      this.recordMetaChangedHandler
+    )
+  }
+
+  compute ([modelName, recordId]) {
```

Comment:
Wouldn't it make more sense to pass in an actual record to the helper?

Reply:
That's also an option indeed, but then I would do the same I think for `setRecordMeta` and `getRecordMeta` on the service I think. I used this approach because it's the same as `ember-data-record-permissions`. We could also change it inside that package. Usage becomes slightly more compact.

Reply:
It's not a requirement, I was just wondering if that might be better. Is there an easy way to get the model name from the record (inside the template). How would it look like with the current implementation?

Something like this just feels redundant:
```
{{record-meta "user" userRecord.id}}
```

Reply:
Okay and what about:
`setRecordMeta(modelName, recordId, recordMeta)` vs `setRecordMeta(record, recordMeta)`
and
`getRecordMeta(modelName, recordId)` vs `getRecordMeta(record)`

Reply:
It doesn't bother me since JS code tends to not be as "noisy" as template code. Maybe it should be done the same to be consistent?

Reply:
Hmm, I don't have an actual record at the time of normalization (which is logic), So I guess the public API of the service must remain as is. Lemme double check.

Reply:
Unless I'd use an additional method for normalization but that might be confusing/overkill?

Reply:
Nah, maybe just keep everything as-is then?

Reply:
The helper now accepts an actual record.